### PR TITLE
TRA-4602 Traffic statistics - Pie Chart

### PR DESCRIPTION
**Pull request by lirazyehezkel**
image

image

### CODE REVIEW

**Comment by gadotroee** [comment]
waiting for https://github.com/up9inc/mizu/pull/1144

File: ui-common/src/components/modals/TrafficStatsModal/TrafficStatsModal.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+interface TrafficStatsModalProps {
+  isOpen: boolean;
+  onClose: () => void;
+  getTrafficStatsDataApi: () => Promise<any>
+}
+
+export const TrafficStatsModal: React.FC<TrafficStatsModalProps> = ({ isOpen, onClose, getTrafficStatsDataApi }) => {
+
+  const modes = Object.keys(StatsMode).filter(x => !(parseInt(x) >= 0));
```
Comment (AmitUp9) [inline ui-common/src/components/modals/TrafficStatsModal/TrafficStatsModal.tsx:35]: Why do you need to filter StatsMode like that?

Comment (lirazyehezkel) [reply ui-common/src/components/modals/TrafficStatsModal/TrafficStatsModal.tsx:35]: Object.keys of an enum return the string keys but also return numerical keys.
for example if we have the enum {a: a, b: b}
Object.keys(enum) will return [a,b,0,1] and we want to filter those numbers

https://www.crojach.com/blog/2019/2/6/getting-enum-keys-in-typescript

Comment (AmitUp9) [reply ui-common/src/components/modals/TrafficStatsModal/TrafficStatsModal.tsx:35]: ok got it. thanks

**Review by AmitUp9 — APPROVED** [review]

**Pull request merged**